rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -503,7 +503,7 @@ void emu_settings::EnhanceSpinBox(QSpinBox* spinbox, SettingsType type, const QS
 	spinbox->setRange(min, max);
 	spinbox->setValue(val);
 
-	connect(spinbox, QOverload<const QString &>::of(&QSpinBox::valueChanged), [=](const QString&/* value*/)
+	connect(spinbox, QOverload<const QString &>::of(&QSpinBox::textChanged), [=](const QString&/* value*/)
```

Comment:
might as well replace "value" with "text"

Reply:
Actually changed to the non-deprecated variant of `valueChanged`, the one using an `int`.